packages/coinstac-simulator/config/default.json

```diff
@@ -0,0 +1,13 @@
+{
+  "backend": false,
```

Comment:
Definitely thought about this when making the configuration. We’re only using it for pdbs right now, so I left it at the root. I’ll go back and adjust.